Source/src/WixSharp.Samples/Wix# Samples/AllInOne/MyAppWix.cs

```diff
@@ -68,9 +68,10 @@ static public void Main()
             // Optionally enable an ability to repair the installation even when the original MSI is no longer available.
             project.EnableResilientPackage();
 
-            // Uncomment to optionally enable the full UI for "Uninstall/Change" button in the Control Panel.
-            project.EnableUninstallFullUI("[#myapp_exe],0");
-
+            // Uncomment one of the following to optionally enable the full UI for "Uninstall/Change" button in the Control Panel.
+            // project.EnableUninstallFullUI("[#myapp_exe],0", @"[TempFolder]CustomMsiLog.log", MsiexecLogSwitches.Star | MsiexecLogSwitches.V);
```

Comment:
That's an interesting fact that [TempFolder] MSI property works here, probably it gets evaluated by MSI later during execution.